package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "notifier",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Write tests for your implementation capturing scenarios such as user happy path and unhappy path, db connection success and failure. Consider using cucumber-js with gherkin to implement BDD.